### DISCUSSION TITLE
Welcome to chickynoid Discussions!

### DISCUSSION THREAD

**Discussion by MrChickenRocket** [easy-games/chickynoid#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by User9684** [comment]
this reminds me of luanoid

**Reply by BrookenRecord** [reply]
Luanoid is similar in that it implements a replacement for Humanoid physics (and rendering?). However, Chickynoid is fundamentally a different project with a different scope - it is server authoritative.

**Comment by szymon227G** [comment]
Are dynamic player sizes possible with chickynoid? (For example: crouching under a low ceiling.) If they're not possible, will they be possible in the future?

**Comment by MrChickenRocket** [comment]
Not possible by default, but it’s also not too difficult a change. The caching in the collision detection just has to support keying to the box size, too.If roblox release fast swept collisions and we switch to them, it also would support it there, then. From: Szymon GraczSent: Sunday, October 16, 2022 3:23 AMTo: easy-games/chickynoidCc: MrChickenRocket; AuthorSubject: Re: [easy-games/chickynoid] Welcome to chickynoid Discussions! (Discussion #1) Are dynamic player sizes possible with chickynoid? (For example: crouching under a low ceiling.) If they're not possible, will they be possible in the future?—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
